recipes/xerces-c/all/conanfile.py

```diff
@@ -67,9 +67,9 @@ def layout(self):
 
     def requirements(self):
         if "icu" in (self.options.transcoder, self.options.message_loader):
-            self.requires("icu/72.1", run=can_run(self))
+            self.requires("icu/73.2", run=can_run(self))
         if self.options.network_accessor == "curl":
-            self.requires("libcurl/7.88.1")
+            self.requires("libcurl/8.2.1")
```

Comment:
```suggestion
            self.requires("libcurl/[>=7.78.0 <9]")
```

Reply:
Let's avoid this until we have an automated way to move every recipe to ranges at once :)

Reply:
@RubenRBS, the doc shall probably updated ? [version-ranges](https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges)